### PR TITLE
Credits

### PR DESCRIPTION
**Pull request by wilhelmy**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,74 @@
+The original Ion was written and maintained by Tuomo Valkonen between 2003 and
+2009, some parts date back to 1999.
+
+After he stopped using Linux, Arnout Engelen took over the maintainership and
```
Comment (raboof) [inline AUTHORS]: Fun fact: @gwash and 'TimAndAHaf' started Notion before I joined, possibly more. Perhaps:

```suggestion
After he stopped using Linux, the community took over the maintainership and
```

File: AUTHORS [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+renamed Ion to Notion to conform to the Ion license, which did not allow for
+the name Ion to be used by forks. The license has since been changed back to
+the ordinary LGPL v2.1+ and code which is only available under the Ion has been
```
Comment (raboof) [inline AUTHORS]:
```suggestion
the ordinary LGPL v2.1+ and code which was only available under the Ion license has been
```

**Comment by wilhelmy** [comment]
Squash and merge?

**Comment by wilhelmy** [comment]
Should we install it along with README.md?

**Comment by raboof** [comment]
> Should we install it along with README.md?

Fine by me, WDYT?

**Comment by knixeur** [comment]
makes sense

**Pull request merged**